prov/verbs/src/fi_verbs.c

```diff
@@ -665,12 +665,18 @@ static int fi_ibv_read_params(void)
 
 static void fi_ibv_fini(void)
 {
+#if HAVE_VERBS_DL
+	ofi_monitor_cleanup();
+#endif
 	fi_freeinfo((void *)fi_ibv_util_prov.info);
 	fi_ibv_util_prov.info = NULL;
 }
 
 VERBS_INI
 {
+#if HAVE_VERBS_DL
+	ofi_monitor_init();
```

Comment:
Doesn't this already happen in src/fabric.c fi_ini? I'm probably missing context here.

Reply:
This is needed when building the verbs provider as a separate DLL.